functions/zoom-downloader.py

```diff
@@ -521,18 +552,36 @@ def s3_filename(self):
 
     def valid_mp4_file(self):
         # TODO: if file not found, add appropriate error
-        s3 = boto3.client("s3")
-        url = s3.generate_presigned_url(
+        url = self.s3.generate_presigned_url(
             "get_object",
             Params={"Bucket": ZOOM_VIDEOS_BUCKET, "Key": self.s3_filename}
         )
 
-        command = ["/var/task/ffprobe", "-of", "json", url]
-        if subprocess.call(command) == 1:
+        cmd = f"/var/task/ffprobe -of json -show_format {url}"
```

Comment:
We were also not getting any useful data out of ffprobe!
This `-show_format` flag gets us some useful output.